Tools/WebKitTestRunner/InjectedBundle/atspi/AccessibilityUIElementAtspi.cpp

```diff
@@ -456,6 +456,13 @@ JSRetainPtr<JSStringRef> AccessibilityUIElement::currentStateValue() const
     return OpaqueJSString::tryCreate(!value.isNull() ? value : "false"_s).leakRef();
 }
 
+JSRetainPtr<JSStringRef> AccessibilityUIElement::sortDirection() const
+{
+    m_element->updateBackingStore();
+    auto value = m_element->attributes().get("sort"_s);
+    return OpaqueJSString::tryCreate(value).leakRef();
```

Comment:
I think it's better to check the value here and convert it to what tests expect (`AXAscendingSortDirection`, `AXDescendingSortDirection` or `AXUnknownSortDirection`). This way we don't have to change every test and results can be shared.